articles/scopes/current/index.md

```diff
@@ -1,23 +1,55 @@
 ---
-title: Scopes
-description: Overview of scopes
+title: Permissions
+description: Understand the principle of permissions (scopes) and explore general examples of their use.
 topics:
   - scopes
+  - permissions
 contentType:
-  - reference
   - concept
   - index
 useCase:
   - development
+  - add-login
+  - secure-api
+  - call-api
 ---
-# Scopes
+# Permissions
 
-The OAuth 2.0 protocol is a delegated authorization mechanism, where an application requests access to resources controlled by the user (the resource owner) and hosted by an API (the resource server). The authorization server issues the application a more restricted set of credentials than those of the user in the form of an Access Token.
+Different pieces of user information are often stored across a number of online resources. Users may upload and store photos with a service like Flickr, keep digital files on Dropbox, and store contacts and events in Google Calendar or on Facebook.
 
-The permissions represented by the Access Token in OAuth 2.0 terms are known as scopes. The `scope` parameter allows the application to express the desired scope of the access request. The `scope` parameter can also be used by the authorization server in the response to indicate which scopes were actually granted (if they are different than the ones requested).
+Often, new applications will want to make use of the information that has already been created in an online resource. To do so, the application must ask for authorization to access this information on a user's behalf. Permissions (also known as _scopes_ in the OAuth 2.0 specs) define the specific actions third-party applications can be allowed to do on a user's behalf.
 
-You can use scopes to:
+## Ways to use permissions
 
-- Let an application verify the identity of a user (by using [Open ID Connect](/protocols/oidc)) and get basic profile information about the user, such as their email or picture. For details, refer to [OpenID Connect Scopes](/scopes/current/oidc-scopes).
+When an app requests permission to access a resource through an authorization server, it uses the `scope` parameter to specify what type of permission it needs, and the authorization server uses the `scope` parameter to respond with the type of permission that was actually granted (if the granted permission was different from what was requested).
```

Comment:
Agreed. This will be a change we make during doc updates specifically related to the Authz release for when it comes out of beta. This doc will go out prior to that, so we can't make those changes just yet.